tests/controllers/home.test.js

```diff
@@ -2,20 +2,10 @@
 
 const AppConstants = require("../../app-constants");
 const home = require("../../controllers/home");
-const { getExperimentBranch } = require("../../controllers/utils");
```

Comment:
Removing the tests for the function as its' not longer included in the home controller.

Reply:
Is there any harm in leaving the tests in place to make sure we don't break the function? We might use it in the future, so would be nice to make sure we don't break it between now and then.

Reply:
Added tests back, and included a language-specific one that should fail in the requested language is not in the first/primary position of the `accept-language` headers.